config/routes.rb

```diff
@@ -2,6 +2,7 @@
   resources :books, only: [:index]
   resources :authors, only: [:index]
 
+  root 'books#index'
```

Comment:
@Alexneisc I'm confused. I don't remember the reason why I have this rule. And I'm not sure if this rule correct.